### PR TITLE
Reworks facehugged effects to make it harder to ignore

### PR DESCRIPTION
**Pull request by zzzmike**
# About the pull request

This makes the paralyze chance when facehugged apply earlier in the hugged phase, and it has a higher chance of happening, but paralyzes for less time when it does happen. Paralyze chance gets slightly higher with a slightly longer time the longer you stay hugged. Changes paralyze message slightly to clue new players into seeking help.

# Explain why it's good for the game

When you're hugged, it should be scary and urgent to deal with, to make the game more interesting and time-sensitive - rather than "*yawn* time for surgery in awhile, I guess, after a bit more fighting." I know balance changes are controversial and I'm very open to changing this PR, but I wanted to at least start a conversation by opening this. My goal is not to nerf marines (if marines get buffed in some other way in a different PR, great) I just want to make the game more interesting and fun.

# Changelog
:cl:
balance: when facehugged, more paralyze chance with a smaller paralyze duration
ui: during hugged effects, changes messaging to encourage new players to seek help 
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm [hunk 1]
```diff
@@ -89,7 +89,7 @@
 			if(stage < 5)
 				counter += 1 * hive.larva_gestation_multiplier
 
-		if(stage < 5 && counter >= 90)
+		if(stage < 5 && counter >= 84)
```
Comment (Zonespace27) [inline code/modules/mob/living/carbon/xenomorph/Embryo.dm:92]: Make this into a define please (the 84), magic numbers suck

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm [hunk 2]
```diff
@@ -98,6 +98,14 @@
 
 	switch(stage)
 		if(2)
+			if(prob(1))
+				if(affected_mob.knocked_out < 1)
+					affected_mob.pain.apply_pain(PAIN_CHESTBURST_WEAK)
+					affected_mob.visible_message(SPAN_DANGER("\The [affected_mob] starts shaking uncontrollably!"), \
```
Comment (Zonespace27) [inline code/modules/mob/living/carbon/xenomorph/Embryo.dm:104]:
```suggestion
					affected_mob.visible_message(SPAN_DANGER("[affected_mob] starts shaking uncontrollably!"), \
```

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm [hunk 3]
```diff
@@ -111,22 +119,30 @@
...
 				affected_mob.pain.apply_pain(PAIN_CHESTBURST_WEAK)
 				var/message = pick("Your chest hurts badly", "It becomes difficult to breathe", "Your heart starts beating rapidly, and each beat is painful")
 				message = SPAN_WARNING("[message].")
 				to_chat(affected_mob, message)
 				if(prob(50))
 					affected_mob.emote("scream")
+			if(prob(4))
+				if(affected_mob.knocked_out < 1)
+					affected_mob.pain.apply_pain(PAIN_CHESTBURST_WEAK)
+					affected_mob.visible_message(SPAN_DANGER("\The [affected_mob] starts shaking uncontrollably!"), \
```
Comment (Zonespace27) [inline code/modules/mob/living/carbon/xenomorph/Embryo.dm:141]:
```suggestion
					affected_mob.visible_message(SPAN_DANGER("[affected_mob] starts shaking uncontrollably!"), \
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm (hunk 1 above)
Comment (zzzmike) [reply code/modules/mob/living/carbon/xenomorph/Embryo.dm:92]: Beagle is moving burst timer to config so I'm happy to just delete this change

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm (hunk 2 above)
Comment (zzzmike) [reply code/modules/mob/living/carbon/xenomorph/Embryo.dm:104]: I agree. (This is just a holdover from the existing paralyze section anyways)

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm (hunk 3 above)
Comment (zzzmike) [reply code/modules/mob/living/carbon/xenomorph/Embryo.dm:141]: agreed as per the other section

File: code/modules/mob/living/carbon/xenomorph/Embryo.dm (hunk 1 above)
Comment (zzzmike) [reply code/modules/mob/living/carbon/xenomorph/Embryo.dm]: now it should match 3844

**Comment by zzzmike** [comment]
(I think it makes sense to leave this as draft until https://github.com/cmss13-devs/cmss13/pull/3844 is merged)

**Comment by morrowwolf** [comment]
Remind me to TM Monday please

**Pull request merged**